### PR TITLE
Add Idaho Permanent Building Fund Tax

### LINKED ISSUE #3179: Add Idaho Permanent Building Fund Tax
The absence of this small Idaho tax makes all the test cases have a tax difference in the TAXSIM35 integration testing described in issue #993.   What do people think about whether or not this tax should be considered part of the Idaho state income tax?

Screenshot 2023-10-20 at 4 01 29 PM

### PR DESCRIPTION
**Pull request by mylittlebecca**
Fixes #3179

### Summary
🏛️🛡️🧮

This pull request adds the Idaho Permanent Building Fund (PBF) tax to the policy engine, along with its parameters, variables, and tests. The PBF tax is a flat tax of $10 per tax unit, with exemptions based on income, age, filing status, and other criteria. The pull request creates new parameter files in `policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/`, new variable files in `policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/`, and new test files in `policyengine_us/tests/policy/baseline/gov/states/id/tax/income/other_taxes/pbf/`. The pull request also updates the references and metadata for the new parameters and variables.

> _`Idaho PBF tax`_
> _New files for parameters, tests_
> _Autumn leaves fall fast_

### Walkthrough
*  Add new parameters and variables for the Idaho Permanent Building Fund (PBF) tax, which is a flat tax of $10 per tax unit with some exemptions based on income, age, filing status, tanf, and blindness. ([link](https://github.com/PolicyEngine/policyengine-us/pull/3235/files?diff=unified&w=0#diff-01b02594f1271d8f59341a1b7c6a787491a45600fada1c081dd7fd76ec89c4a6L1-R14), [link](https://github.com/PolicyEngine/policyengine-us/pull/3235/files?diff=unified&w=0#diff-e9930680502708b465152533be2e979a761610d66dcfdeea30be70596f1505c1L1-R14), [link](https://github.com/PolicyEngine/policyengine-us/pull/3235/files?diff=unified&w=0#diff-65d8aecc62ad6b5eea82fc971a71b6eaf4f0b772fc01b13bf100e19e1d56cd4bL1-R30), [link](https://github.com/PolicyEngine/policyengine-us/pull/3235/files?diff=unified&w=0#diff-c4c123af5fca0495e4c9f7658a23f8351eb20e37e5b66d50c443399e4a80dbf0L1-R30), [link](https://github.com/PolicyEngine/policyengine-us/pull/3235/files?diff=unified&w=0#diff-84003effc66390e218db751a7f2baa4848a7669f72d2bbf1e2a3d28d0d6bb3d2R1-R31), [link](https://github.com/PolicyEngine/policyengine-us/pull/3235/files?diff=unified&w=0#diff-4cc5f15c4b8a04dbebbc87db3690e84c02cb48ad55e9c38502844d58e6287d14L1-R17), [link](https://github.com/PolicyEngine/policyengine-us/pull/3235/files?diff=unified&w=0#diff-474c727709c79c22a3181bc5154740153cefb0851f700711b0120851e194d10eL1-R41))
*  Add new test files for the `id_pbf` and `id_pbf_eligibility` variables, which check the tax amount and the exemption criteria respectively. ([link](https://github.com/PolicyEngine/policyengine-us/pull/3235/files?diff=unified&w=0#diff-67ec7f83939fad200cdce55ca0c9769945740e17e2bdb3cb46244c58ad58250fL1-R6), [link](https://github.com/PolicyEngine/policyengine-us/pull/3235/files?diff=unified&w=0#diff-110ff3a52ed4bc0f93a45f104ad13c5115b67bf1c5625a676c5f6351a5fb62e8R1-R30))

### CODE REVIEW

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/amount.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/amount.yaml:1]: need a changelog_entry.yaml

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/amount.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+description: Idaho requires filers to pay PBF tax of this amount.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/amount.yaml:1]:
```suggestion
description: Idaho requires filers to pay permanent building tax of this amount.
```

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/amount.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+values:
+  2021-01-01: 10
+
+metadata:
+  label: Idaho PBF tax amount
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/amount.yaml:6]:
```suggestion
  label: Idaho permanent building tax amount 
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/amount.yaml:6]: same in all labels and descriptions

File: policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from policyengine_us.model_api import *
+
+
+class id_pbf(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf.py:7]:
```suggestion
    label = "Idaho permanent building tax"
```

File: policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf_eligibility.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
+from policyengine_us.model_api import *
+
+
+class id_pbf_eligibility(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Idaho PBF tax exemption credit"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf_eligibility.py:7]:
```suggestion
    label = "Idaho permanent building tax payment required"
```

File: policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf_eligibility.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
+from policyengine_us.model_api import *
+
+
+class id_pbf_eligibility(Variable):
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf_eligibility.py:4]:
```suggestion
class id_pbf_requirement(Variable):
```

File: policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf_eligibility.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf_eligibility.py:1]: rename to `id_pbf_requirement.py`

File: policyengine_us/tests/policy/baseline/gov/states/id/tax/income/other_taxes/pbf/id_pbf_eligibility.yaml
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/id/tax/income/other_taxes/pbf/id_pbf_eligibility.yaml:1]: rename to `id_pbf_requirement.yaml`

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/elderly_age.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/elderly_age.yaml:1]: rename this to age_threshold

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_partial_65_and_above.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_partial_65_and_above.yaml:1]: all in the `pdf` folder - make a folder called `income_threshold` with files `younger`/ + folder `older` in which we have `one_spouse` / `two_spouses`

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_65_and_above.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+# Both spouses
+JOINT: 
+  2021-01-01: 27_800 
+  2022-01-01: 28_700
+HEAD_OF_HOUSEHOLD:
+  2021-01-01: 20_500
+  2022-01-01: 21_150
+SEPARATE:
+  2021-01-01: 5
+  2022-01-01: 5
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_65_and_above.yaml:28]:
```suggestion
```

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_partial_65_and_above.yaml [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+# One spouses
+JOINT: 
+  2021-01-01: 26_450 
+  2022-01-01: 27_300
+HEAD_OF_HOUSEHOLD:
+  2021-01-01: 20_500
+  2022-01-01: 21_150
+SEPARATE:
+  2021-01-01: 5
+  2022-01-01: 5
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_partial_65_and_above.yaml:28]:
```suggestion
```

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_under_65.yaml [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+# Both spouses
+JOINT: 
+  2021-01-01: 25_100 
+  2022-01-01: 25_900
+HEAD_OF_HOUSEHOLD:
+  2021-01-01: 18_800
+  2022-01-01: 19_400
+SEPARATE:
+  2021-01-01: 5
+  2022-01-01: 5
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_under_65.yaml:28]:
```suggestion
```

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/elderly_age.yaml [hunk 9]
```diff
@@ -0,0 +1,15 @@
+description: Idaho considers tax units with head or spouse this age or older to be elderly in exempt from PBF tax.
+values:
+  2021-01-01: 65
+
+metadata:
+  label: Idaho elderly age threshold used in PBF tax
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/elderly_age.yaml:6]:
```suggestion
  label: Idaho permanent building tax age threshold
```

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/elderly_age.yaml [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+  period: year
+  unit: year
+  reference:
+  - title: Idaho Individual Income Tax General Information 2022.
+    href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_03-01-2023.pdf#page=10
+  - title: Idaho Individual Income Tax General Information 2021.
+    href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_11-15-2021.pdf#page=10
+  - title: ID Admin Code 35.01.01.855 PERMANENT BUILDING FUND TAX
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/elderly_age.yaml:14]:
```suggestion
  - title: ID Admin Code 35.01.01.855 PERMANENT BUILDING FUND TAX (2)
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/elderly_age.yaml:14]: we have if there is a tax code part that references the age threshold and income threshold

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/elderly_age.yaml [hunk 11]
```diff
@@ -0,0 +1,15 @@
+description: Idaho considers tax units with head or spouse this age or older to be elderly in exempt from PBF tax.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/elderly_age.yaml:1]:
```suggestion
description: Idaho exempts tax units with higher income from the permanent building tax with filers or spouses at or above this age threshold.
```

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_65_and_above.yaml [hunk 12]
```diff
@@ -0,0 +1,31 @@
+description: Idaho provides PBF exemption for filers of age 65 and above less than this income, depending on filing status.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_65_and_above.yaml:1]:
```suggestion
description: Idaho exempts tax units with two older filers from the permanent building fund tax with income at or below this amount, depending on filing status.
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_65_and_above.yaml:1]: all three descriptions

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_65_and_above.yaml [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+metadata:
+  label: Idaho PBF threshold of Age 65 and above
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_65_and_above.yaml:3]:
```suggestion
  label: Idaho permanent building fund tax two older spouses income threshold
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_65_and_above.yaml:3]: all three labels

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_65_and_above.yaml [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: Idaho Individual Income Tax General Information 2022.
+    href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_03-01-2023.pdf#page=10
+  - title: Idaho Individual Income Tax General Information 2021.
+    href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_11-15-2021.pdf#page=10
+  - title: ID Admin Code 35.01.01.855 PERMANENT BUILDING FUND TAX
+    href: https://regulations.justia.com/states/idaho/35/35-01-01/section-35-01-01-855/
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/threshold_65_and_above.yaml:14]: need legal code reference

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/age_threshold.yaml [hunk 15]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  label: Idaho permanent building tax age threshold
+  period: year
+  unit: year
+  reference:
+  - title: Idaho Individual Income Tax General Information 2022.
+    href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_03-01-2023.pdf#page=10
+  - title: Idaho Individual Income Tax General Information 2021.
+    href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_11-15-2021.pdf#page=10
+  - title: Idaho Individual Income Tax Basics - Income requirement for filing
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/age_threshold.yaml:14]: I don't think that this source supports any of our parameters

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/age_threshold.yaml:14]: we need a legal code

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/age_threshold.yaml [hunk 16]
```diff
@@ -0,0 +1,15 @@
+description: Idaho exempts tax units with higher income from the permanent building tax with filers or spouses at or above this age threshold.
+values:
+  2021-01-01: 65
+
+metadata:
+  label: Idaho permanent building tax age threshold
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/age_threshold.yaml:6]:
```suggestion
  label: Idaho permanent building fund tax age threshold
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/age_threshold.yaml:6]: use permanent building fund tax everywhere

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/age_threshold.yaml [hunk 17]
```diff
@@ -0,0 +1,15 @@
+description: Idaho exempts tax units with higher income from the permanent building tax with filers or spouses at or above this age threshold.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/age_threshold.yaml:1]:
```suggestion
description: Idaho exempts filers from the permanent building fund tax at or above this age threshold.
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/age_threshold.yaml:1]: use permanent building fund tax everywhere

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/threshold_65_and_above.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/threshold_65_and_above.yaml:1]: rename to two_aged

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/threshold_partial_65_and_above.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/threshold_partial_65_and_above.yaml:1]: rename to one_aged

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/younger/threshold_under_65.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/younger/threshold_under_65.yaml:1]: just make a younger.yaml - dont need folder

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/younger/threshold_under_65.yaml [hunk 18]
```diff
@@ -0,0 +1,30 @@
...
+    href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_03-01-2023.pdf#page=10
+  - title: Idaho Individual Income Tax General Information 2021.
+    href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_11-15-2021.pdf#page=10
+  - title: Idaho Individual Income Tax Basics - Income requirement for filing
+    href: https://tax.idaho.gov/taxes/income-tax/individual-income/online-guide/
+
+SINGLE:
+  2021-01-01: 12_550
+  2022-01-01: 12_950
+# Both spouses
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/younger/threshold_under_65.yaml:19]:
```suggestion
```

File: policyengine_us/tests/policy/baseline/gov/states/id/tax/income/other_taxes/pbf/id_pbf_requirement.yaml [hunk 19]
```diff
@@ -0,0 +1,30 @@
...
+    age_head: 60
+    age_spouse: 63
+    tanf: 0
+    blind_head: false
+    blind_spouse: false
+    state_code: ID
+  output:
+    id_pbf_requirement: false
+
+
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/id/tax/income/other_taxes/pbf/id_pbf_requirement.yaml:30]: delete extra line

File: policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf_requirement.py [hunk 20]
```diff
@@ -0,0 +1,50 @@
+from policyengine_us.model_api import *
+
+
+class id_pbf_requirement(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Idaho PBF tax exemption credit"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf_requirement.py:7]:
```suggestion
    label = "Required to pay the Idaho personal building fund tax "
```

File: policyengine_us/tests/policy/baseline/gov/states/id/tax/income/other_taxes/pbf/id_pbf.yaml
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/id/tax/income/other_taxes/pbf/id_pbf.yaml:1]: make one more false test

File: policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf_requirement.py [hunk 21]
```diff
@@ -0,0 +1,50 @@
...
+        # joint_status = filing_status.possible_values.JOINT
+        # widow_status = filing_status.possible_values.WIDOW
+        # dependent_count = tax_unit("tax_unit_dependents", period)
+
+        elder_head = tax_unit("age_head", period) >= p.age_threshold
+        elder_spouse = tax_unit("age_spouse", period) >= p.age_threshold
+        income_ineligible_check = where(
+            elder_head | elder_spouse,
+            income
+            < p.income_threshold.older.threshold_partial_65_and_above[
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf_requirement.py:28]: could you separate the conditions out and comment?

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/amount.yaml [hunk 22]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  label: Idaho permanent building fund tax amount 
+  period: year
+  unit: currency-USD
+  reference:
+  - title: Idaho Individual Income Tax General Information 2022.
+    href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_03-01-2023.pdf#page=10
+  - title: Idaho Individual Income Tax General Information 2021.
+    href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_11-15-2021.pdf#page=10
+  - title: ID Code § 57-1110 (2022)
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/amount.yaml:14]:
```suggestion
  - title: ID Code § 57-1110 
```

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/two_aged.yaml [hunk 23]
```diff
@@ -0,0 +1,30 @@
...
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: Idaho Individual Income Tax General Information 2022.
+    href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_03-01-2023.pdf#page=10
+  - title: Idaho Individual Income Tax General Information 2021.
+    href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_11-15-2021.pdf#page=10
+  # The income threshold is in the Resident Filing Requirement and not in the legale code
+  - title: Idaho Admin. Code r. 35.01.01.855 - PERMANENT BUILDING FUND TAX
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/two_aged.yaml:14]: add the filing requirement legal code reference

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/one_aged.yaml [hunk 24]
```diff
@@ -0,0 +1,31 @@
+description: Idaho exempts tax units with one older filer from the permanent building fund tax with income at or below this amount, depending on filing status and married filing jointly.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/one_aged.yaml:1]:
```suggestion
description: Idaho exempts tax units with at least one older filer from the permanent building fund tax with income at or below this amount, depending on filing status and married filing jointly.
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/one_aged.yaml
[the remark this answers is not in the document]
Comment (MaxGhenis) [reply policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/one_aged.yaml:1]: Where does it say that? I just see that you're required to pay it if you're required to file taxes, i.e. if your income exceeds the standard deduction. But you can just check that tax before credits > 0

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/one_aged.yaml [hunk 25]
```diff
@@ -0,0 +1,32 @@
+description: Idaho exempts tax units with at least one older filer from the permanent building fund tax with income at or below this amount, depending on filing status and married filing jointly.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/one_aged.yaml:1]: delete the income threshold files as discussed

File: policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/one_aged.yaml [hunk 26]
```diff
@@ -0,0 +1,32 @@
...
+  label: Idaho permanent building fund tax minimum one older spouse income threshold
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/id/tax/income/other_taxes/pbf/income_threshold/older/one_aged.yaml:2]:
```suggestion
metadata:
  label: Idaho permanent building fund tax minimum one older spouse income threshold
```

File: policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf_requirement.py [hunk 27]
```diff
@@ -0,0 +1,54 @@
...
+        # eligible if receiving public assistance tanf
+        tanf_received = spm_unit("tanf", period)
+        tanf_ineligible = tanf_received > 0
+
+        # eligible if head or spouse is blind
+        blind_head = tax_unit("blind_head", period)
+        blind_spouse = tax_unit("blind_spouse", period)
+        blind_ineligible = blind_head | blind_spouse
+
+        return ~income_ineligible & ~tanf_ineligible & ~blind_ineligible
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf_requirement.py:54]: dont need the income eligible test

**Comment by PavelMakarchuk** [comment]
Duplicate #3480

**Pull request closed without merging**